pkg/daemon/ceph/osd/volume.go

```diff
@@ -810,13 +812,22 @@ func GetCephVolumeRawOSDs(context *clusterd.Context, clusterInfo *client.Cluster
 		osdID := osdInfo.OsdID
 
 		if osdInfo.CephFsid != cephfsid {
-			logger.Infof("skipping osd.%d: %s running on a different ceph cluster %q", osdID, osdInfo.OsdUUID, osdInfo.CephFsid)
+			message := fmt.Sprintf("osd.%d: %q running on a different ceph cluster %q", osdID, osdInfo.OsdUUID, osdInfo.CephFsid)
```

Comment:
Users might benefit from the cluster name as well as the FSID in the case of a k8s cluster with multiple Rook-Ceph clusters.

Reply:
This message will be in the osd prepare log, which is specific to a cluster namespace, so the cluster name shouldn't be needed in this case.

Reply:
But isn't this complaining that the OSD belongs to a **different** cluster? It would be helpful to note which other cluster is owning the disk and not just by FSID. If possible.